### ISSUE TITLE
Setup live reload option in Electron

### ISSUE THREAD

**Issue by slaweet** [LiskHQ/lisk-nano#899]
### Expected behaviour
There should be a way to `npm run start` (maybe with a param) to serve the React app from webpack dev server with live reload:
```
win.loadURL('http://localhost:8080/');
```

Also `npm run start` should include the `webpack run build-electron`

### Actual behaviour
`npm run start` serves file from filesystem
https://github.com/LiskHQ/lisk-nano/blob/446c755340b3d5bc7aa71ea433a4dbedb0bf3b8c/app/src/main.js#L74

**Comment by V1shvesh** [comment]
Can I work upon this?

**Comment by slaweet** [comment]
Yes, @V1shvesh, you're welcome to work on this. If you'll have any questions, feel free to ask here or on [Lisk Gitter](https://gitter.im/LiskHQ/lisk)

**Comment by V1shvesh** [comment]
I'm really new to Node. Can I maybe get some pointers?

**Comment by V1shvesh** [comment]
@slaweet Please tell me when can I find you online...

**Comment by slaweet** [comment]
@V1shvesh, in general, I'm online MON-FRI 8AM-5PM CET
Ad your questions:
1. I think you need "start": "npm run build-electron && electron ./app/"
2. By param I mean either to have 
 - two commands `npm run start` (works as now) and `npm run start-live` (runs with http://localhost:8080)
 - or an optional parameter, e.g. `npm run start -- --loadURL=http://localhost:8080`. I don't know exactly how to parse the param, but https://discuss.atom.io/t/how-to-pass-command-arguments-in-electron/17247 could give some hints. The default of `loadURL` should be what it is now.
3. I guess you figured this out yourself in https://github.com/LiskHQ/lisk-nano/pull/937

**Comment by V1shvesh** [comment]
@slaweet thanks a lot!! Sorry to be a bother though, and I'll get to it right away
